example/main.cpp

```diff
@@ -28,12 +28,11 @@ int main(int argc, char *argv[]) {
 
     try {
         renderer = std::make_unique<inexor::vulkan_renderer::Application>(argc, argv);
-    } catch (const inexor::vulkan_renderer::VulkanException &exception) {
+    } catch (const std::exception &exception) {
         spdlog::critical(exception.what());
         std::abort();
     }
```

Comment:
```suggestion
    } catch (const std::exception &exception) {
        spdlog::critical(exception.what());
        return 1;
    }
```

Reply:
Done. I also added this for the validation layer exception handler.